__tests__/app.test.js

```diff
@@ -254,12 +254,50 @@ describe("GET /api/articles/:article_id/comments", () => {
         });
       });
   });
-  test("404 returns Not found when specific id is invalid", () => {
+  test("400 returns Not found when specific id is invalid", () => {
     return request(app)
       .get("/api/articles/9999/comments")
+      .expect(400)
```

Comment:
So if the article_id is a number, it should be valid. In this case, it's a number but there's no article yet assigned to this number. So I'm thinking this should be a 404 instead 😄